src/System.Buffers.Experimental/System/Buffers/Text/BufferWriter_strings.cs

```diff
@@ -0,0 +1,200 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Text.Utf8;
+
+namespace System.Buffers.Text
+{
+    public ref partial struct BufferWriter
+    {
+        #region String
+        public bool TryWrite(string text)
+        {
+            if (!TryWrite(text, out var written)) return false;
+            _written += written;
+            return true;
+        }
+
+        public void Write(string text)
+        {
+            while (!TryWrite(text)) Resize();
+        }
+
+        public bool TryWrite(string text, TransformationFormat format)
+        {
+            int written;
+            if (!TryWrite(text, out written)) return false;
+            if (!format.TryTransform(Free, ref written)) return false;
+            _written += written;
+            return true;
+        }
+
+        public void Write(string text, TransformationFormat format)
+        {
+            while (!TryWrite(text, format)) Resize();
+        }
+
+        public bool TryWriteLine(string text)
+        {
+            if (!TryWriteLine(text, out var written)) return false;
+            _written += written;
+            return true;
+        }
+
+        public void WriteLine(string text)
+        {
+            while (!TryWriteLine(text)) Resize();
+        }
+
+        public bool TryWriteLine(string text, TransformationFormat format)
+        {
+            int written;
+            if (!TryWriteLine(text, out written)) return false;
+            if (!format.TryTransform(Free, ref written)) return false;
+            _written += written;
+            return true;
+        }
+
+        public void WriteLine(string text, TransformationFormat format)
+        {
+            while (!TryWriteLine(text, format)) Resize();
+        }
+        #endregion
+
+        #region UTF8String
+        public bool TryWrite(Utf8Span text)
+        {
+            int written = text.Bytes.Length;
+            if (!text.Bytes.TryCopyTo(Free)) return false;
+            _written += written;
```

Comment:
We don't need `written` local here. Why not just do:
```C#
_written += text.Bytes.Length;
```

Reply:
will fix